code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -410,10 +410,20 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	return ""
 
 /datum/gas_mixture/react(turf/open/dump_location)
+	var/static/list/nonreactive_gases = list(typecacheof(/datum/gas/oxygen, /datum/gas/nitrogen, /datum/gas/carbon_dioxide)) // These gases cannot react amongst themselves
 	. = NO_REACTION
+	if(gases.len)

```

Comment:
presumably you mean `if(!gases.len)`

Reply:
That would explain why things suddenly got even faster...